docs/integrations/microsoft-azure/azure-virtual-machine.md

```diff
@@ -0,0 +1,155 @@
+---
+id: azure-virtual-machine
+title: Azure Virtual Machine
+description: Learn about the Sumo Logic collection process for the Azure Virtual Machine service.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/microsoft-azure/azure-vm-icon.png')} alt="Thumbnail icon" width="50"/>
+
+
+[Azure Virtual Machine](https://learn.microsoft.com/en-us/azure/virtual-machines/) is a virtualized computing environment on Microsoft's cloud platform, Azure, that allows users to run applications and tasks on a dedicated, scalable, and on-demand basis. Azure VM provides scalable computing capacity in the Azure Cloud. You can use Azure VM to launch as many or as few virtual servers as you need, configure security and networking, and manage storage.
+
+The Sumo Logic app for Azure Virtual Machine allows you to collect your VM instance metrics and display them using predefined dashboards. The app provides dashboards to display analysis of VM instance metrics for CPU, disk, network, cache, and memory. Also, it provides detailed insights into all audit events associated with VM instances and specifically helps identify changes, errors, and user activities.
+
+
+
+## Log and metric types
+
+For Azure Virtual Machine, you can collect the following logs and metrics:
+
+* **Activity logs**. To learn more about the different Activity log category types and schemas collected for Azure Virtual Machine, refer to [Azure documentation](https://learn.microsoft.com/en-us/azure/virtual-machines/monitor-vm-reference#activity-log).
+* **Virtual Machine Metrics**. These metrics are available in [Microsoft.Compute/virtualMachines](https://learn.microsoft.com/en-us/azure/virtual-machines/monitor-vm-reference#metrics) namespace.
+
+For more information on supported metrics and their units, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/virtual-machines/monitor-vm-reference#supported-metrics-for-microsoftcomputevirtualmachines).
+
+## Setup
+
+Azure service sends monitoring data to Azure Monitor, which can then [stream data to Eventhub](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/stream-monitoring-data-event-hubs). Sumo Logic supports:
+
+* Logs collection from [Azure Monitor](https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-get-started) using our [Azure Event Hubs source](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
+* Metrics collection using our [HTTP Logs and Metrics source](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor/) via Azure Functions deployed using the ARM template.
```

Comment:
This should be same as old Azure Key Vault app where we added the beta metrics source description